portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -9,10 +9,10 @@
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
-
+    String msg = "Toya!"; 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("<h1> Hello " + msg + "</h1>");
```

Comment:
Looks good! I noticed there's an extra space in front of the `Hello` which you'll see in the suggestion below:

```suggestion
    response.getWriter().println("<h1>Hello " + msg + "</h1>");
```